streamer/src/streamer.rs

```diff
@@ -313,7 +313,7 @@ impl StreamerSendStats {
 }
 
 impl StakedNodes {
-    pub fn new(stakes: Arc<HashMap<Pubkey, u64>>, overrides: HashMap<Pubkey, u64>) -> Self {
+    pub fn new(stakes: Arc<HashMap<Pubkey, u64>>, overrides: Arc<HashMap<Pubkey, u64>>) -> Self {
```

Comment:
Why do we suddenly need to have an Arc here?

Reply:
It was an attempt to update it in try_refresh_stake_info which was called in a loop. I changed the StakedNodes to support updating the stake map. There is no need to update the overrides now. It is now set once in the new constructor.